GNUmakefile

```diff
@@ -231,6 +240,23 @@ $(rust_src_dir)/%_stemmer.rs: algorithms/%/stem_Unicode.sbl snowball
 	echo "./snowball $< -rust -o $${o}"; \
 	./snowball $< -rust -o $${o}
 
+$(go_src_main_dir)/stemwords/algorithms.go:
+	@echo "Generating algorithms.go"
+	@cd go/stemwords && go generate && cd ../..
```

Comment:
`cd ../..` isn't useful here - each command in a make rule is run in its own shell (or effectively is).